.cirrus.yml

```diff
@@ -114,6 +115,7 @@ build_task:
               CI_DESIRED_NETWORK: cni
               CI_DESIRED_DATABASE: boltdb
               CI_DESIRED_STORAGE: vfs
+              ALLOW_LEGACY_DB: true
```

Comment:
As an FYI (I don't know it to be a problem yet): Adding any differentiation element to a matrix item will result in Cirrus forcing a unique (and often unreadable) task name.  It only does this if `name` is not already unique among the matrix.  That's why we have the `&std_name_fmt` YAML alias.  It (previously) results in mostly human-readable task names.

So, it may be worth considering if adding `$ALLOW_LEGACY_DB` to that alias might be helpful to people trying to sift through 91-some-odd checks.

Reply:
It should be set everywhere CI_DESIRED_DATABASE is set to "boltdb".  I mostly wanted a new variable for readability.

Reply:
Just looked through a fistful of tasks.  You're right, it seems to be okay. The Cirrus-CI Name-uniqueness `$DIETY` is fully satisfied :grin: